BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAppSessionManager.m

```diff
@@ -78,11 +85,26 @@ - (id)init
     self = [super init];
     if (self != nil) {
         _userIdToAssociateIdAndSessionTaskInfo = [NSMutableDictionary new];
+        [self setUpCache];
         [self populateSessionTaskIdMap];
     }
     return self;
 }
 
+- (void)setUpCache
+{
+    NSString *cachesPath = [BOXSampleAppSessionManager urlRequestCache];
+    BOOL isDir = NO;
+    if ([[NSFileManager defaultManager] fileExistsAtPath:cachesPath isDirectory:&isDir] == NO || isDir == NO) {
```

Comment:
is `fileExistsAtPath` is true, doesn't it not matter if it's a directory or not? The path matched, so either way there will be a conflict